### PR TITLE
Jquery refactor

### PR DESCRIPTION
**Pull request by RogerDMK**
# Description
Removed remaining XMLHTTP requests, changing it to jquery.
closes#80 

Fixes # (issue)
Fixed a registration bug

### CODE REVIEW

File: src/public/fetchMode.js [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-function setMode() {
-  const response = JSON.parse(this.responseText)
-  const darkMode = response.recordset[0].isDarkmode
-  document.cookie = 'darkMode='+darkMode+'; path=/'
-  window.location.href = '/loginRedirect'
+function getMode (username) {
+  $.get('/user/api/DarkModeData', { usernameInput: username })
+    .done(function (response) {
+      const darkMode = response.recordset[0].isDarkmode
+      document.cookie = 'darkMode=' + darkMode + ';'
```
Comment (RobynGebbie) [inline src/public/fetchMode.js:7]: Specify path is "/"

Comment (RogerDMK) [reply src/public/fetchMode.js]: Added the change!

File: src/public/fetchMode.js [hunk 2]
```diff
@@ -1,17 +1,13 @@
 'use strict'
 
-const new_request = new XMLHttpRequest()
```
Comment (RobynGebbie) [inline src/public/fetchMode.js:3]: request is used in another file, since it's defined here the other file can't see it

Comment (Ray-R4y) [reply src/public/fetchMode.js:3]: request should not be used , we not using xmlHTTPRequest

Comment (RobynGebbie) [reply src/public/fetchMode.js:3]: It gives an error and the password change does not work

File: src/views/login.html [hunk 3]
```diff
@@ -11,8 +11,9 @@
     <script defer type="text/javascript" src="/src/public/utils.js"></script>
     <script defer type="text/javascript" src="/src/public/fetchMode.js"></script>
     <script src="/src/public/login.cjs"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jsencrypt/2.3.1/jsencrypt.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/4.1.1/crypto-js.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```
Comment (Ray-R4y) [inline src/views/login.html:16]: import repeated function

**Review by Ray-R4y — CHANGES_REQUESTED** [review]

**Review by Ray-R4y — APPROVED** [review]
Design 
Good changes to design , change from XMLHTTP to Jquery, which made code beating and more to the style guide.
functionality.
code function very well 
complexity
complexity of the code reduced changing to jquery
naming and style
naming and style follows the style guide
good things
the usage of jquery is so much better than xmlHTTP.

File: src/public/fetchMode.js (hunk 2 above)
Comment (RogerDMK) [reply src/public/fetchMode.js:3]: This request is not

**Pull request merged**